library/src/main/java/com/xwray/groupie/GroupieViewHolder.java

```diff
@@ -94,4 +95,19 @@ public Item getItem() {
     public View getRoot() {
         return itemView;
     }
+
+    public ItemDetailsLookup.ItemDetails<Long> getItemDetails() {
```

Comment:
Not sure whether this is the best place for this. Should this instead be on the item so that users can overwrite it? I can imagine us wanting to provide users with a custom selection key.

Reply:
Regarding the type of the custom selection key I would like to suggest Long.